pubspec.yaml

```diff
@@ -10,7 +10,8 @@ environment:
 
 dependencies:
   async: '>1.9.0 <3.0.0'
-  protobuf: ^0.14.0
+  pedantic: ^1.8.0
+  protobuf: ^1.0.0
```

Comment:
Do we need to bump the minimum version here or can use `>=0.14.0 <2.0.0` here?

Also do we need to regenerate the proto files?

Reply:
I need to update `protobuf` dep to 1.0.0 to be compatible with [`appengine` version 0.10.3](https://pub.dev/packages/appengine).

Any hints on how I should re-generate the proto files? Do we have a script somewhere?

Reply:
> I need to update `protobuf` dep to 1.0.0 to be compatible with [`appengine` version 0.10.3](https://pub.dev/packages/appengine).

That shouldn't have anything to do with the lower bound here though - as long as our upper bound supports it you should be fine.

> Any hints on how I should re-generate the proto files? Do we have a script somewhere?

Unfortunately no I don't, we never went through figuring out how to build these files externally. The process has typically been to build the file internally where its all handled for us and then just copy the file out. However that won't work right now because proto is still at 0.14.x internally.

@sigurdm or @nichite can maybe point us in the right direction for how to generate these files externally?

Reply:
If I set my `appengine` dep to ^0.10.3 in `dart-services`, the pub solver has a fit:

```
[dart-services] pub get
Resolving dependencies...
Because no versions of bazel_worker match >0.1.21 <0.1.22 and bazel_worker >=0.1.22 depends on protobuf ^0.14.0, bazel_worker >0.1.21 requires protobuf ^0.14.0.
And because bazel_worker 0.1.21 depends on protobuf >=0.10.4 <0.14.0 and appengine >=0.10.3 depends on protobuf ^1.0.0, appengine >=0.10.3 is incompatible with bazel_worker >=0.1.21.
So, because dart_services depends on both bazel_worker ^0.1.21 and appengine ^0.10.3, version solving failed.
exit code 1
```

So, no, ^0.14.0 is not compatible ^1.0.0.

Reply:
done